charts/linkerd-control-plane/templates/destination.yaml

```diff
@@ -235,6 +235,9 @@ spec:
             path: /ping
             port: 9996
           initialDelaySeconds: 10
+          {{- if (.Values.destinationController.livenessProbe).timeoutSeconds }}
+          timeoutSeconds: {{ .Values.destinationController.livenessProbe.timeoutSeconds }}
+          {{- end }}
```

Comment:
A more succinct way of writing this would be:
```yaml
          {{- with (.Values.identity.livenessProbe).timeoutSeconds }}
          timeoutSeconds: {{ . }}
          {{- end }}
```

Reply:
Thanks, refactored.